src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/domainEvents/OutboundEventsPublisher.kt

```diff
@@ -26,9 +28,10 @@ class OutboundEventsPublisher(
   }
   private val domainEventsTopicClient by lazy { domainEventsTopic.snsClient }
 
-  fun publishDomainEvent(event: HMPPSDomainEvent, licenceId: String) {
+  @TransactionalEventListener(phase = TransactionPhase.AFTER_COMMIT)
+  fun publishDomainEvent(event: HMPPSDomainEvent) {
     val eventType = event.eventType
-    log.debug("Event {} for licence ID {}", eventType, licenceId)
+    log.debug("Event {} for licence ID {}", eventType, event.additionalInformation?.licenceId)
```

Comment:
We're generating duplicate logs in app insights for this - 2 for each event raised as we're not filtering out `debug`.. maybe it would be worth consolidating these messages into one and log it before the event is raised? 

The log line in the `also` doesn't really have enough contextual info
There should also be a `dependency` that would confirm the event is raised if investigating an issue